.github/workflows/client.yml

```diff
@@ -108,6 +108,16 @@ jobs:
           tags: ${{ env.GCR_REGISTRY_URL }}/${{ env.GOOGLE_PROJECT_ID }}/${{ env.IMAGE_NAME }}
           labels: revision=${{ github.sha }}
           push: ${{ github.event_name == 'workflow_dispatch' }}
+
+      - # Temp fix - move cache instead of copying (added below step and
+        # modified value of `cache-to`).
+        # https://github.com/docker/build-push-action/issues/252
+        # https://github.com/moby/buildkit/issues/1896
+        # Without the change some jobs were failing with `no space left on device`
+        name: Move cache
```

Comment:
Let's move it to be the very last step in workflows.

Reply:
Done, see ec61e35f2ff5f1ea6f51c631464e94be2ee0c1e8.